Project.toml

```diff
@@ -13,9 +13,9 @@ StatsBase = "2913bbd2-ae8a-5f71-8c99-4fb6c76f3a91"
 [compat]
 Distances = "^0.9,^0.10"
 MLJModelInterface = "^0.3.5,^0.4, 1.0"
-MultivariateStats = "0.9"
+MultivariateStats = "0.9.1"
```

Comment:
Do we really need to make this 0.9.1? Generally, unless we really need it, compat specs do not restrict to a particular patch.